### ISSUE TITLE
OL 7.7: cli installed from "quick start" installation script requires pyton3 but won't use it.

### ISSUE THREAD

**Issue by aorcl** [oracle/oci-cli#239]
Hello, 
Installing on new Oracle Linux 7.7 VM

Doing the "quick start" installation from bash script linked in https://docs.cloud.oracle.com/iaas/Content/API/SDKDocs/cliinstall.htm 

The CLI installs but every run it nags that the python 3 is required

```
$ oci os ns get
WARNING: OCI CLI now requires Python 3.5+. Please install or upgrade your version of Python to 3.5+ before February 13, 2020 to avoid interruption to CLI usage.
{
  "data": "zzzz"
}
```

But even after installation of python3 `sudo yum install python3` the CLI doesn't pick it up and continue throwing the warning.

**Comment by vish1** [comment]
@aorcl We are looking into this issue internally and will provide an update soon.

**Comment by rakesh-253** [comment]
I am also facing the same issue.

bash ~>python
Python 2.7.5 (default, Jun 20 2019, 13:30:36) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36.0.1)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 
bash ~>python3
Python 3.6.8 (default, Aug  7 2019, 08:02:28) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-39.0.1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>
 bash ~> oci os ns get
WARNING: OCI CLI now requires Python 3.5+. Please install or upgrade your version of Python to 3.5+ before February 13, 2020 to avoid interruption to CLI usage.
{
  "data": "xxxxxxxx"
}

**Comment by lucianf** [comment]
Fixed in 2.9.2 (#249).

**Comment by vish1** [comment]
As pointed out by @lucianf, this issue has now been fixed.
